.github/workflows/build.yml

```diff
@@ -9,12 +9,23 @@ jobs:
   build:
     name: Build
     runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        working-directory: Statiq.Web
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
     steps:
       - name: Get Source
         uses: actions/checkout@v2
+        with:
+          path: Statiq.Web
+
+      - name: Checkout Statiq.Framework
+        uses: actions/checkout@v2
+        with:
+          repository: statiqdev/Statiq.Framework
+          path: Statiq.Framework
```

Comment:
Interesting, is the idea here that this'll make sure builds are successful even if Statiq Web relies on unreleased parts of Statiq Framework and has `<LocalReferences>true</LocalReferences>` in the root `Directory.Build.props` (as is often the case between releases). If so, this is awesome.

Reply:
Saw the comment above which confirms, this is a great way to handle it for the Actions verification build since it's often in a state where it's using pre-release functionality.